Podfile

```diff
@@ -19,6 +24,7 @@ target 'swiftgen' do
 end
 
 target 'SwiftGenKit' do
+  common_pods
   podspec :path => 'SwiftGenKit.podspec'
```

Comment:
Now that we have `common_pods`, `SwiftGenKit` should be moved into `common` too, right?